mito-ai/src/Extensions/AiChat/ChatMessage/ChatDropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ExpandedVariable } from './ChatInput';
+
+interface ChatDropdownProps {
+    options: ExpandedVariable[];
+    selectedIndex: number;
+    onSelect: (variableName: string, parentDf?: string) => void;
+}
+
+const ChatDropdown: React.FC<ChatDropdownProps> = ({
+    options,
+    selectedIndex,
+    onSelect,
+}) => {
+    const getShortType = (type: string) => {
+        return type.includes("DataFrame") ? "df"
+            : type.includes("<class '") ? type.split("'")[1]
+                : type;
+    }
+
+    return (
+        <ul className="chat-dropdown-list">
+            {options.map((option, index) => {
+                const uniqueKey = option.parent_df
+                    ? `${option.parent_df}.${option.variable_name}`
+                    : option.variable_name;
+
+                return (
+                    <li
+                        key={uniqueKey}
+                        className={`chat-dropdown-item ${index === selectedIndex ? 'selected' : ''}`}
+                        onClick={() => onSelect(option.variable_name, option.parent_df)}
+                    >
+                        <span className="chat-dropdown-item-type"
+                            style={{
+                                color: getShortType(option.type) === 'df' ? 'blue'
+                                    : getShortType(option.type) === 'col' ? 'orange'
+                                        : "green"
+                            }}
+                        >
+                            {getShortType(option.type)}
+                        </span>
+                        <span className="chat-dropdown-item-name">
+                            {option.variable_name}
+                        </span>
+                        {option.parent_df && (
```

Comment:
Sweet! displaying the dataframe name is so needed. Look at this example I got while testing
<img width="349" alt="Screenshot 2024-11-14 at 7 33 24 PM" src="https://github.com/user-attachments/assets/b4f655c6-dda9-4c2b-bf70-71e181483523">